docs/arch/adr-0001-style-guide.md

```diff
@@ -0,0 +1,76 @@
+# Style Guide
+
+
+## Context
+
+We need a style guide for our actions repo. This structure provides the following benefits:
+- developers can spend more time on solving the problem
+- PR reviews are limited to discussing intent
+- format related feedback can point to a collective style guide instead of personal preferences
+- consumers know how to find what they're looking for
+- developers know where to look when fixing something
+
+## Options
+
+This style guide is a collection of individual options. Instead of providing permutations of these options,
+this ADR provides a section for an initial proposal for several style guide topics and a section
+that contains the agreed upon formats. This ADR should be updated to remove reference to the initial
+proposal prior to merging in order to avoid confusion.
+
+As an alternative, each topic could be its own ADR. Reviewers may request this approach, which will require
+breaking this ADR into several ADRs and associated PRs.
```

Comment:
We should split this into workflows and actions.  They're different.

Reply:
So we would have one style guide for workflows and another for actions? There's a significant overlap. If we want to distinguish between actions and workflows, but reduce redundancy, can we do one shared style guide, and then an action style guide and workflow style guide that only focus on things specific to actions/workflows?